### DISCUSSION TITLE
Feature Request

### DISCUSSION THREAD

**Discussion by M3hran** [Cybrarist/Discount-Bandit#35] [Ideas]
Hi Cybrarist, 
First, Thank you for this awesome tool. Secondly, I was not sure where to ask for feature requests so I decided to put it here, hoping you will see it here. 

1-Stores: It would be great if you can add hardware and auto shop stores to the list. In particular:

*** Home Depot
*** Lowes
*** Harbor Freight
*** Advanced Auto
*** AutoZone

2- Google Sheets Sync: The ability to export to Google Sheets with continuous updates.

3- Gotify Notification.  

Thanks & Keep up the good work.

**Comment by Cybrarist** [comment]
Hi @M3hran 

1- sure ill add them to the todo list but there's no proper timeline tbh unless someone creates a PR for them.

2- theres no point to have a sheet with live updates since that's the point of the app, but a PR is welcome and i'll merge it.

3- apprise is already implemented and it supports gotify already.

**Reply by M3hran** [reply]
hey @Cybrarist 

1- Understood. Thanks!

2- Understood, the app is great. syncing with google sheets allows other calculations, collaboration, and reporting that will expand the usability/productivity imho. [ also import from google sheets for long lists of products/urls ]

3- Awesome, Thanks!

**Reply by Cybrarist** [reply]
Hi @M3hran

i will happily merge it if someone is willing to put the effort to implement this feature.
